site2/docs/admin-api-transactions.md

```diff
@@ -0,0 +1,710 @@
+---
+id: admin-api-transactions
+title: Manage transactions
+sidebar_label: "Transactions"
+---
+
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+````
+
+
+> **Important**
```

Comment:
To enhance readability, is it better to update it to a `Tip` as Tip will be highlighted while `Important` not?

Reply:
Yep, I thought it could be batch fixed and aligned in this issue #15892. Didn't notice it has been addressed for other pages :)
I will align the change on this new page as well.